zeppelin-web/src/components/modal-shortcut/modal-shortcut.html

```diff
@@ -15,292 +15,297 @@
 <div class="modal fade" id="shortcutModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
   <div class="modal-dialog">
     <div class="modal-content">
-      <div class="modal-header">
+      <div class="shortcut-modal-header">
         <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
```

Comment:
Can we make `x` button more visible?

![image](https://cloud.githubusercontent.com/assets/4968473/25282386/444ae248-26eb-11e7-820f-0d9a72b1b55a.png)

![image](https://cloud.githubusercontent.com/assets/4968473/25282399/4d518eaa-26eb-11e7-87f2-64f7268153da.png)